app/main.py

```diff
@@ -5,8 +5,9 @@
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi_utils.tasks import repeat_every
 
-from app.db import insert_data, load_data
-from app.scraper import deduplicate, frankenbert_rank, scrape_twitter
+
+from app.db import Database
```

Comment:
Why import Database from db.py as well as an instance of it from scraper.py? Maybe I'm missing something, but the former appears unused in this file.